src/store/index.js

```diff
@@ -32,11 +36,39 @@ export default {
 
     /**
      * Sets the terms of use accepted state.
-     * @param {Object} - the current state
-     * @param {Boolean} - true if the terms of use are accepted
+     * @param {Object} state - the current state
+     * @param {Boolean} value - true if the terms of use are accepted
      */
     acceptTermsOfUse(state, value) {
       state.termsOfUseAccepted = value;
+    },
+
+    /**
+     * Sets the quality score to the specified value.
+     * @param {Object} state - the current state.
+     * @param {Number} score - the quality score to set.
+     */
+    setQualityScore(state, score) {
+      state.qualityScore = score;
+    }
+  },
+
```

Comment:
Whereas mutations are synchronous and update the store directly, actions may be asynchronous and update the store via committing a mutation.